### ISSUE TITLE
Server accessibility

### ISSUE THREAD

**Issue by julmot** [hstove/issue_stats#41]
The reason I switched over to issuestats.com instead of isitmaintained.com was because the server was not always available, the load time of repository pages was incredibly low as badge loading times were slow.

Unfortunately I'm experiencing the same issues on issuestats.com. Since a few days the site is not available for me, or not all assets will be loaded:

![2016-03-03_171242](https://cloud.githubusercontent.com/assets/8310677/13500552/592e09ca-e163-11e5-96ad-77facbfa9ef5.png)

Often the error looks like:
![2016-03-03_171411](https://cloud.githubusercontent.com/assets/8310677/13500553/592f0d52-e163-11e5-987f-c8ac32e2f2da.png)

So I'm asking you Hank @hstove, what server technology is used for issuestats.com? Maybe we can find a solution for a better availability.

**Comment by julmot** [comment]
@hstove Seems like you are uninterested in your own service. Will think about removing it from all of my repos and recommend this also to others.

**Comment by hstove** [comment]
@julmot indeed, this has always been a free and open source project that was primarily built for Github's data challenge. It was never meant to be a 'startup' or 'service'. I sadly just haven't had the time to maintain it lately. I have no problem with you recommending isitmaintained over issuestats, it seems like a great project and is also currently maintained.

I appreciate you using issuestats!
